taskcluster_util/taskcluster_download.py

```diff
@@ -150,12 +151,15 @@ def run(self):
             # no artifact_name, then get the latest artifacts list
             self.show_latest_artifacts(task_id)
         elif self.should_display_signed_url_only is True:
-            print self.artifact_downloader.get_signed_url(task_id, self.artifact_name)
+            signed_url = self.artifact_downloader.get_signed_url(task_id, self.artifact_name)
+            print(signed_url)
+            return signed_url
         else:
             # has artifact_name, then download it
-            logger.info('Downloading latest artifact [{}] of TaskID [{}] ...'.format(self.artifact_name, task_id))
+            logger.info('Downloading [{}] from TaskID [{}] ...'.format(self.artifact_name, task_id))
             local_file = self.artifact_downloader.download_latest_artifact(task_id, self.artifact_name, self.dest_dir)
-            logger.info('Download [{}] from TaskID [{}] to [{}] done.'.format(self.artifact_name, task_id, local_file))
+            logger.debug('Download to [{}]'.format(local_file))
+            return local_file
```

Comment:
[Like here](https://github.com/askeing/taskcluster-util-python/pull/30/files#r49172105), I'm not sure to understand.